packages/api/models/schemas.js

```diff
@@ -620,6 +620,13 @@ module.exports.provider = {
       title: 'Host',
       type: 'string',
     },
+    allowedRedirects: {
+      title: 'Allowed hosts for redirect when retrieving data. If a redirect outside of this list is encountered, data retrival will fail.',
+      type: 'array',
+      items: {
+        type: 'string',
+      },
+    },
```

Comment:
So when a user creates a new Provider via the dashboard, they will be able to add these? Or is this obscured from users because they don't really need to specify and we know the current hosts?

What happens with existing providers? How do they get `allowedRedirects`?

This might be easier as a quick chat.

Reply:
Checked with the OX team and it seems like the provider edit form int he dashboard automatically reads the Dynamo schema and should add a field for editing it. However, I'm not sure how that's going to handle the `allowedRedirects` field which is an array type, so I'm gonna test it myself.

Reply:
![Screen Shot 2021-07-28 at 11 55 16 AM](https://user-images.githubusercontent.com/1001694/127502331-81421720-83a7-49b6-85e4-5b4b9c7000bf.png)

The dashboard actually handled it great by default